README.md

```diff
@@ -1,4 +1,101 @@
 # eosdactoolkit
-The DAC toolkit for building and running DACs
 
-[Design document](https://docs.google.com/document/d/1C4yzFNpK0Iz0Ru0gz28HeLJic5vZWBmVl3wV8czhVS4/edit#)
+## Purpose
+The DAC toolkit for building and running DACs. This is the member client where the DAC members will interface with the deployed smart contract code. The code is written in javascript using Vue.js and Quasar-framework.js.
+
+## Pre-Requisites
+You want to be able to read javascript code and have a basic understanding of frontend web development. If you want to simply change the assets to your DAC's messaging, branding, and images then the processes is not too difficult. The details are in `eosdac-material/README.md`
+
+
+
+[Original Design document](https://docs.google.com/document/d/1C4yzFNpK0Iz0Ru0gz28HeLJic5vZWBmVl3wV8czhVS4/edit#).
+
+# Design Document
```

Comment:
I do not recommend adding the design document to the readme. It was helpful during the initial design, but has no place in a readme document.